addons/account/wizard/account_payment_register.py

```diff
@@ -497,16 +497,21 @@ def _compute_payment_method_line_fields(self):
     @api.depends('payment_type', 'journal_id')
     def _compute_payment_method_line_id(self):
         for wizard in self:
-            if wizard.journal_id:
-                available_payment_method_lines = wizard.journal_id._get_available_payment_method_lines(wizard.payment_type)
+            move_payment_method_line_id = wizard.line_ids.move_id.preferred_payment_method_line_id
```

Comment:
1) you could have multiple moves here. Use `partner_id` instead? Or manage this directly in the batching key?
2) don't suffix a variable by `_id` if it's not an id but a record! :angry:

Reply:
The payment method line on the move has priority, and actually it works even if we have multiple moves. 
Ex: If i go on the list view of invoices and select 2 differents invoices with different partner and so different payment method line
![image](https://github.com/user-attachments/assets/1276f708-848a-440b-9f55-1f9e81f70f87)
![image](https://github.com/user-attachments/assets/2bdfb00e-be3d-41cd-b3c7-e9b45482505d)

So just rename the variable is enough i think ?